twitter.js

```diff
@@ -49,9 +62,68 @@ class Twitter {
     };
 
     this.url = getUrl(config.subdomain);
+    this.oauth = getOauth(config.subdomain);
     this.config = config;
   }
+  
+  async reqTkn(callback) {
+    // return `oauth_token` & `oauth_token_secret`
+    // https://developer.twitter.com/en/docs/basics/authentication/api-reference/request_token
+    
+    const requestData = {
+      url: `${this.oauth}/request_token`,
+      method: "POST"
+    };
+    
+    var parameters = {};
+    if(callback) parameters = { "oauth_callback": callback };
+    if (parameters) requestData.url += "?" + querystring.stringify(parameters);
+
+    let headers = {};
+    headers = this.client.toHeader(
+      this.client.authorize(requestData, {})
+    );
 
+    const results = await Fetch(requestData.url, {
+      method: "POST",
+      headers: Object.assign({}, baseHeaders, headers)
+    })
+    .then(res => res.text())
+    .then(txt => formatRes(txt));
+    
+    return results;
+  }
+  
+  async accTkn(key, secret, verifier) {
```

Comment:
do you think we should use keyword args here? 

```es6
async getAccessToken({ key, secret, verifier }) {
}
```

That way the user will not confuse which parameter goes where?